ner4opt/utils.py

```diff
@@ -19,6 +19,13 @@
 pluralizer = inflect.engine()
 
 
+def preprocess(text: str) -> str:
```

Comment:
so this does not break anything from the past? 

Also, please add a few unit tests. Add a unit test for the actual issue, drop a comment on issue number and say that this unit test comes from that issue. Add other unit test to show that this new preprocess() works, and a few tests for edge cases, empty input etc, if not tested already

Reply:
Yes, I checked if all our previous tests are passing, they are. 

Good idea, I will add a test for this too.